website/docs/d/resource_pool.html.markdown

```diff
@@ -41,10 +41,10 @@ data "vsphere_resource_pool" "pool" {
 }
 ```
 
-### Using with ESXi
+### Using with Standalone ESXi
 
-On ESXi, you don't have to specify either attribute to use this data source. An
-empty declaration will load the default resource pool.
+When using ESXi without vCenter, you don't have to specify either attribute to
+use this data source. An empty declaration will load the default resource pool.
```

Comment:
from a UI / consistency point of view for this kind of information - this may probably make more sense as a `NOTE` i.e.:

```
~> **Note: Usage with Standalone ESXi** When using ESXi without vCenter, you don't have to specify either attribute to use this data source. An empty declaration will load the default resource pool.
```

Reply:
I like this - will make the change!